app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,50 @@
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
```

Comment:
- [ ] We should have a sign-in method. [Requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/sign_in_method_requirement.png).